bleak/backends/bumble/utils.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2024 Victor Chavez
+from bumble.core import UUID
+
+
+def bumble_uuid_to_str(uuid: UUID) -> str:
+    """
+    Convert a native Bumble UUID to a string representation.
+    Reason for this conversion is that the string representation (__str__) done in bumble
+    is not a standard UUID string representation.
+    In addition, the byte representation from data member uuid_bytes of bumble.core.UUID
+    is represented in reverse order.
+    Example:
+        Shortened 16 bit UUID: 0x1800Bumble UUID class
+        bumble UUID string representation: 'UUID-16:1800 (Generic Access)'
+        bumble UUID bytes representation: b'\x00\x18'
+    """
```

Comment:
This docstring isn't getting parsed correctly, needs to be updated to conform to google spec.